packages/sources/cfbenchmarks/src/endpoint/crypto.ts

```diff
@@ -48,34 +47,22 @@ export type BaseEndpointTypes = {
   Response: SingleNumberResultResponse
 }
 
-export const additionalInputValidation = ({ index, base, quote }: Params): void => {
-  // Base and quote must be provided OR index must be provided
-  if (!(index || (base && quote))) {
-    const missingInput = !index ? 'index' : 'base /or quote'
-    throw new AdapterInputError({
-      statusCode: 400,
-      message: `Error: missing ${missingInput} input parameters`,
-    })
-  }
-}
-
 export const cryptoRequestTransform = (
   req: AdapterRequest<typeof inputParameters.validated>,
   settings: BaseEndpointTypes['Settings'],
 ): void => {
-  // TODO: Move additional input validations to proper location after framework supports it
-  additionalInputValidation(req.requestContext.data)
-  if (!req.requestContext.data.index) {
+  const { base, quote, index } = req.requestContext.data
```

Comment:
do you think it would be worth it to refactor and extract this part of the change since it is similar as in lwba?